elm/src/Faq/Update.elm

```diff
@@ -0,0 +1,21 @@
+module Faq.Update exposing (..)
+
+import Faq.Types exposing (Model, Msg(..))
+import Return exposing (Return, singleton)
+
+init : Model
+init =
+    { topic = ""
+    , title = "FAQ - Flint"
+    , selectedTopic = 0
+    }
+
+
+update : Msg -> Model -> Return Msg Model
+update msg model =
+    case msg of
+        Select topicId ->
+            singleton { model | selectedTopic = topicId }
```

Comment:
The indentation is sometimes 2 spaces sometimes 4. You can use a formatter here.

Reply:
I just updates some with default 4 space indentation and made new commit

Reply:
Ok, thank you! I think more work is needed for this pr. I wrote some requested changes, please check them out :)